runconfig/parse_test.go

```diff
@@ -322,10 +322,10 @@ func TestParseDevice(t *testing.T) {
 			PathInContainer:   "/something",
 			CgroupPermissions: "rwm",
 		},
-		"/dev/snd:/something:ro": {
+		"/dev/snd:/something:rw": {
 			PathOnHost:        "/dev/snd",
 			PathInContainer:   "/something",
-			CgroupPermissions: "ro",
+			CgroupPermissions: "rw",
 		},
```

Comment:
I think this unit test is for testing if devices option works. For validate part, that's in opts/opts_test.go:TestValidateDevice(), I added two invalid cases about `rr`, is that ok?